src/Tribe/Service_Providers/Context.php

```diff
@@ -270,6 +270,38 @@ static function ( $post_type ) {
 					Tribe__Context::REQUEST_VAR => [ TEC::TAXONOMY ],
 				],
 			],
+			'view_url' => [
+				'read'  => [
+					Tribe__Context::FUNC => [
+						static function () {
+							return Utils\View::get_data( 'url', Tribe__Context::NOT_FOUND );
+						},
+					],
+					Tribe__Context::LOCATION_FUNC => [
+						// Handles the datepicker request.
+						'view_data',
+						static function ( $data ) {
+							$date_k = 'tribe-bar-date';
+							return is_array( $data ) && isset($data[ $date_k ])
```

Comment:
* :no_entry_sign: Expected 1 spaces after opening bracket; 0 found
* :no_entry_sign: Expected 1 spaces before closing bracket; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: 8739c95042e83ffb1deaa575ac25a0b9</sub>